src/utils/prepareBlocks.js

```diff
@@ -0,0 +1,78 @@
+const Message = require('../factories/Message');
+const { createHeadingSection, createFooterSection, createEmptyRelease } = require('../factories/Blocks/section');
+
+/**
+ * Slack has a 50 block limit, per message. We need to chunk blocks so we can send them
+ * in separate messages if there are more than 50 blocks in a message.
+ *
+ * More here: https://api.slack.com/reference/block-kit/blocks
+ * @param {Object} heading heading section
+ * @param {Array} blocks team blocks to be inserted
+ * @param {Object} footer footer section
+ */
+const chunkBlocks = (heading, blocks, footer) => {
+  const allBlocks = [heading, ...blocks, footer];
+
+  if (allBlocks.length <= 50) return [allBlocks];
+
+  let counter = 0;
+  const chunkedBlocks = blocks.reduce(
+    (acc, block) => {
+      if (counter < 49) {
+        counter += 1;
+        const lastIndex = acc.length - 1;
+        acc[lastIndex].push(block);
+        return acc;
+      }
+
+      counter = 0;
+      return [...acc, [block]];
+    },
+    [[]],
+  );
```

Comment:
I see we already depend on `lodash`, do you think its `chunk` function would help here? https://lodash.com/docs#chunk

Reply:
Yeah, this looks great! I did not know this existed.

So the array number will just be messages.length / 49 (one to unshift and push the heading/footer) and then Math.ceil. Then it will sort itself out. Yeah that is perfect! Thank you!